app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -0,0 +1,31 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-signin-code-header">{{#t}}Authorize this sign-in{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div class="graphic graphic-mail">{{#t}}Authorize this sign-in{{/t}}</div>
+
+    <p class="verification-email-message">
+    {{#t}}Check your email for the code sent to %(email)s.{{/t}}
+    </p>
+
+    <form novalidate>
+      <div class="input-row token-code-row">
+        <input type="text" class="tooltip-below" id="token_code" placeholder="{{#t}}Enter code{{/t}}" required autofocus />
+      </div>
+
+      <div class="button-row">
+        <button type="submit" class="use-logged-in">{{#t}}Continue{{/t}}</button>
+      </div>
+
+    </form>
+
+    <div class="links">
+        <a href="{{{ escapedSupportLink }}}" id="support-link" class="delayed-fadein right" target="_blank">{{#t}}Why is this happening?{{/t}}</a>
```

Comment:
![image](https://user-images.githubusercontent.com/128755/34850206-ef164e06-f6f2-11e7-98b3-b69d19d9ecb8.png) links to https://support.mozilla.org/en-US/kb/accounts-blocked , is that correct? Seems like there is no info there for this particular view?

Reply:
Hrm, I probably should have looked at linked site. I thought it was a generic `enter code received` to login. Tracking this [here](https://github.com/mozilla/fxa-content-server/issues/5830).